.kokoro/release/publish-docs.cfg

```diff
@@ -0,0 +1,34 @@
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 73713
+      keyname: "yoshi-automation-github-key"
+    }
+  }
+}
+
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 73713
+      keyname: "docuploader_service_account"
+    }
+  }
+}
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Use the trampoline script to run in docker.
+build_file: "nodejs-pubsub/.kokoro/trampoline.sh"
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:8-user"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-pubsub/.kokoro/publish-docs.sh"
```

Comment:
Silly question - what actually calls this script?  I would assume it's part of the release publish script, no?

Reply:
I was planning on chaining this job (publish-docs) to the release publish job. Would it make more sense for the package publishing and docs publishing to happen in one job?

Reply:
Nah, whatever you have planned is fine.  Was just curious :)